versioned/storage/cassandra/src/main/java/org/projectnessie/versioned/storage/cassandra/CassandraConstants.java

```diff
@@ -101,6 +103,10 @@ final class CassandraConstants {
           + COL_REFS_POINTER
           + "=? AND "
           + COL_REFS_DELETED
+          + "=? AND "
+          + COL_REFS_CREATED_AT
```

Comment:
Have you considered the (admittedly contrived) case where 2 distinct updates could succeed if they have the same timestamp, because they were generated by 2 distinct Nessie instances? Wouldn't it be safer to use some random thing, maybe a UUID?